Lib/sqlite3/test/dbapi.py

```diff
@@ -933,6 +935,72 @@ def test_on_conflict_replace(self):
         self.assertEqual(self.cu.fetchall(), [('Very different data!', 'foo')])
 
 
+class MultiprocessTests(unittest.TestCase):
+    def setUp(self):
+        with sqlite.connect(TESTFN) as cx:
+            cx.execute("create table t(t)")
+
+    def tearDown(self):
+        unlink(TESTFN)
+
+    def test_rollback_if_db_is_locked(self):
+        # bpo-27334: ctx manager does not rollback if commit fails
+        TIMEOUT = 0.01
```

Comment:
Just checking: you want to keep this or you missed changing this to `SHORT_TIMEOUT`?

Reply:
I want to keep it. `SHORT_TIMEOUT` is way too long (default 30.0). Perhaps I should rename it to `SQLITE_TIMEOUT`, or `CONNECTION_TIMEOUT` though.

Reply:
Yes, I would prefer to have it in a way we can configure it easily and maybe reuse it.

Reply:
(Um, there was some context missing from my post. Added now.)

Reply:
> Yes, I would prefer to have it in a way we can configure it easily and maybe reuse it.

I agree. Fixed in f67591cc421ef1fcac915bda39edd492c1b38d87

Reply:
PTAL :)